types/azdata/index.d.ts

```diff
@@ -14,5267 +13,5272 @@
-        /**
-         * String containing the full server version text.
-         */
-        serverVersion: string;
-        /**
-         * String describing the product level of the server.
-         */
-        serverLevel: string;
-        /**
-         * The edition of the instance.
-         */
-        serverEdition: string;
-        /**
-         * Whether the instance is running in the cloud (Azure) or not.
-         */
-        isCloud: boolean;
-        /**
-         * The version of Azure that the instance is running on, if applicable.
-         */
-        azureVersion: number;
-        /**
-         * The Operating System version string of the machine running the instance.
-         */
-        osVersion: string;
-        /**
-         * options for all new server properties.
-         */
-        options: { [key: string]: any };
-    }
-
-    /**
-     * The possible values of the server engine edition
-     */
-    export enum DatabaseEngineEdition {
-        Unknown = 0,
-        Personal = 1,
-        Standard = 2,
-        Enterprise = 3,
-        Express = 4,
-        SqlDatabase = 5,
-        SqlDataWarehouse = 6,
-        SqlStretchDatabase = 7,
-        SqlManagedInstance = 8,
-        SqlOnDemand = 11
-    }
-
-    export interface DataProvider {
-        handle?: number;
-        readonly providerId: string;
-    }
-
-    export interface ConnectionProvider extends DataProvider {
-        connect(connectionUri: string, connectionInfo: ConnectionInfo): Thenable<boolean>;
-
-        disconnect(connectionUri: string): Thenable<boolean>;
-
-        cancelConnect(connectionUri: string): Thenable<boolean>;
-
-        listDatabases(connectionUri: string): Thenable<ListDatabasesResult>;
-
-        changeDatabase(connectionUri: string, newDatabase: string): Thenable<boolean>;
-
-        rebuildIntelliSenseCache(connectionUri: string): Thenable<void>;
-
-        getConnectionString(connectionUri: string, includePassword: boolean): Thenable<string>;
-
-        buildConnectionInfo?(connectionString: string): Thenable<ConnectionInfo>;
-
-        registerOnConnectionComplete(handler: (connSummary: ConnectionInfoSummary) => any): void;
-
-        registerOnIntelliSenseCacheComplete(handler: (connectionUri: string) => any): void;
-
-        registerOnConnectionChanged(handler: (changedConnInfo: ChangedConnectionInfo) => any): void;
-    }
-
-    export enum ServiceOptionType {
-        string = 'string',
-        multistring = 'multistring',
-        password = 'password',
-        number = 'number',
-        category = 'category',
-        boolean = 'boolean',
-        object = 'object'
-    }
-
-    export enum ConnectionOptionSpecialType {
-        connectionName = 'connectionName',
-        serverName = 'serverName',
-        databaseName = 'databaseName',
-        authType = 'authType',
-        userName = 'userName',
-        password = 'password',
-        appName = 'appName'
-    }
+	import * as vscode from 'vscode';
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/azdata/index.d.ts;
  Line: 21;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
Thanks, updated